src/Akka.Persistence.MongoDb.Tests/Bug61FixSpec.cs

```diff
@@ -175,7 +175,7 @@ public object ToJournal(object evt)
         }
 
 
-        private static Config CreateSpecConfig(DatabaseFixture databaseFixture, int id)
```

Comment:
so you actually need this Id if the tests are being run in parallel - otherwise you'll get cross-contamination.

Reply:
There has to be a way to segregate tests, each onto their own database.